zipline/pipeline/factors/technical.py

```diff
@@ -144,6 +150,238 @@ def compute(self, today, assets, out, close, volume):
         out[:] = nanmean(close * volume, axis=0)
 
 
+class RollingPearsonOfReturns(SingleInputMixin, CustomFactor):
+    """
+    Calculates the Pearson product-moment correlation coefficient of the
+    returns of the given asset with the returns of all other assets.
```

Comment:
We probably should say somewhere in this doc that this is what most people mean when they just say "correlation coefficient".